### PR TITLE
Adding a Czech section to the Language Resources page

### PR DESCRIPTION
**Pull request by burningtree**
Hi, I added a Czech section to the "Language resources" page, maybe it will be useful for someone. 🇨🇿 

It includes links to our content that we create within @gweicz and a czech translation of the book Mastering Ethereum.

Thank you so much for the PR merge!

### CODE REVIEW

File: src/content/community/language-resources/index.md [hunk 1]
```diff
@@ -68,6 +68,13 @@ If you are bilingual and want to help us reach more people, you can also get inv
 - [SecbitLabs](https://mp.weixin.qq.com/s/69_tqBJpr_sbaKtR1sBRMw) - a WeChat account, explaining cryptography, security, etc.
 - [Sparkbyte](https://mp.weixin.qq.com/s/9KgKTc_jtJ7bWKdbNPoqvQ) - a WeChat account, explaining zk technology
 
+### Czech {#cs}
+
+- [Gwei.cz](https://gwei.cz) - local community around Web3, creates educational content, organises online and in-person events
+- [Gwei.cz Příručka](https://prirucka.gwei.cz/) - Ethereum guide for beginners
+- [DAO Příručka](https://dao.gwei.cz/) - beginner's guide to the DAOs
+- [Mastering Ethereum](https://ipfs.infura-ipfs.io/ipfs/bafybeidvuxhnsgfx3tncpfxheqglkjwmdxclknlgd7s7qggd2a6bzgb27m) - Mastering Ethereum in Czech
```
Comment (minimalsm) [inline src/content/community/language-resources/index.md:76]:
```suggestion
- [Mastering Ethereum](https://ipfs.infura-ipfs.io/ipfs/bafybeidvuxhnsgfx3tncpfxheqglkjwmdxclknlgd7s7qggd2a6bzgb27m) - Mastering Ethereum in Czech
- [Women in Crypto](https://holkyvkryptu.cz/)
```

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks for the PR @burningtree 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @burningtree for docs

**Comment by burningtree** [comment]
@minimalsm  Perfect! I'm already on Discord as tree#7466

Thanks for adding the "Holky v kryptu" - I was considering whether to add them since they are not Ethereum/web3 focused. But they are great for absolute beginners. So why not. 👍

**Pull request merged**